posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -500,9 +505,11 @@ def _counts_breakdown_value(self):
             case WebStatsBreakdown.CITY:
                 return parse_expr("tuple(properties.$geoip_country_code, properties.$geoip_city_name)")
             case WebStatsBreakdown.TIMEZONE:
-                # Timezone offsets would be slightly more useful, but that's not easily achievable
-                # with Clickhouse, we might attempt to change this in the future
-                return ast.Field(chain=["properties", "$timezone"])
+                # Get the difference between the UNIX timestamp at UTC and the UNIX timestamp at the event's timezone
+                # Value is in milliseconds, turn it to hours, works even for fractional timezone offsets (I'm looking at you, Australia)
+                return parse_expr(
+                    "if(isNull(properties.$timezone), NULL, (toUnixTimestamp64Milli(parseDateTimeBestEffort(assumeNotNull(toString(timestamp, properties.$timezone)))) - toUnixTimestamp64Milli(timestamp)) / 3600000)"
```

Comment:
Let's add a test for key not present, value is NULL, and value is an empty string, I think you might need some extra logic to handle the empty string.